### ISSUE TITLE
Stuck on Placing Items (Multiple Builds)

### ISSUE THREAD

**Issue by PepGames** [Z3DR/MM3D_Randomizer#64]
Hey there, currently using the latest nightly build. The build is [Nightly-5d9cf4]. I was also using an older nightly build from 1.2.2 from a few weeks ago (can't remember the hash). Either way, both are giving me the same issue. I am stuck on placing items. I am using a 3DS console and I am on version 1.0 of MM. I can only assume it's due to my options selected. I am running glitchless, all locations reachable 'no', and I have every stray fairy spot and skulltulla spot excluded. I am assuming the issue is due to this? But maybe I'm wrong. 

What do you think?

**Comment by PhlexPlexico** [comment]
It's probably due to your settings. You don't need to exclude any of the locations since the items that are placed there are vanilla. If you remove all the stray fairy and skulltula locations, it should generate.

Thanks!

**Comment by PepGames** [comment]
I should have known... that means we can't mix the skulltullas and stray fairies in then right?

**Comment by PepGames** [comment]
it builds with no logic hehe

**Comment by PhlexPlexico** [comment]
Correct. Skullsanity and Stray Fairies will never be shuffled (as of right now). There will eventually be options to explicitly shuffle, but for now you don't need to worry about it. Even when you generate a seed the in game tracker will show the default items.

**Comment by PepGames** [comment]
Thanks for the clarification!

**Comment by PepGames** [comment]
Okay actually, I removed all the exclusions, it still crashed upon loading a new save file. I am going to try default settings.

**Comment by PhlexPlexico** [comment]
What version are you on, and can you provide the crash log that it shows on the 3ds screen? If you're using [Nightly-446774](https://github.com/Z3DR/MM3D_Randomizer/releases/tag/Nightly-446774), please make sure to update, as there is a known crashing bug if you do not have the lens of truth on a new save.

**Comment by PepGames** [comment]
As I said in the first post the build is [Nightly-5d9cf4]. Let me see if I can grab a crash log for ya, I might have to wait til I get home from work later today to be able to post it here.

**Comment by PepGames** [comment]
Okay. Here's what I have tried. I went into the titles and deleted the patch entirely to bring the game back to its vanilla state (I thought maybe there could have been something stale in the patch or something, this was not the case). 

I booted the game with no patch, it boots, I can start a new save file and get into the game.

I make the settings default on version [Nightly-5d9cf4]: Generate randomizer 

Launch game, crashes. 

I will try 445774!! 

So so sorry about my posts being all over the place regarding this.

**Comment by PepGames** [comment]
Okay, so I installed Nightly 446774, settings default. Generated a patch and the game launches totally fine! 

Thank you! I am not sure why, but github had listed 5d9cf4 above 446774 when I navigated to the releases page and I didn't bother to scroll down to look at the other builds. So this caused me to think 5d9cf4 was the most recent build when in fact, it was not.

I am going to go ahead and close this ticket, as it seems like me having the wrong build was the major issue. I will play around with exclusions later today. Thank you so much for your help.
